File: src/test/java/com/everyonewaiter/domain/notification/DiscordEmbedTest.java
package com.everyonewaiter.domain.notification;

import static com.everyonewaiter.domain.notification.NotificationFixture.createDiscordEmbed;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class DiscordEmbedTest {

  @Test
  void addField() {
    DiscordEmbed embed = createDiscordEmbed();

    embed.addField(new DiscordField("이름", "내용"));

    assertThat(embed.getFields()).hasSize(1);
  }

}

File: src/test/java/com/everyonewaiter/domain/notification/DiscordEmbedTest.kt
package com.everyonewaiter.domain.notification

import org.assertj.core.api.Assertions.assertThat
import kotlin.test.Test

class DiscordEmbedTest {

  @Test
  fun `디스코드 필드 추가`() {
    val embed = createDiscordEmbed()

    embed.addField(DiscordField("이름", "내용"))

    assertThat(embed.fields).hasSize(1)
  }
}
